site/sfguides/src/getting_started_with_native_apps/getting_started_with_native_apps.md

```diff
@@ -538,7 +688,47 @@ Next, you'll load data into these newly defined tables using the Snowflake UI.
 ## Install the Application
 Duration: 3
 
-To use the application, we'll first need to install it in the account. Normally you'd simply click an install button in the Snowflake Marketplace, but since we're building the application and using a single account to demonstrate the provider and consumer experiences, you'll need to run the following SQL to install the application in the account. 
+To use the application, we'll first need to install it in the account. Normally you'd simply click an install button in the Snowflake Marketplace, but since we're building the application and using a single account to demonstrate the provider and consumer experiences, you'll need to either run a Snowflake CLI command or some SQL to install the application in the account. 
```

Comment:
```suggestion
To use the application, we'll first need to install it in the account. Normally you'd simply click an install button in the Snowflake Marketplace, but since we're building the application using a single account to demonstrate the provider and consumer experiences, you'll need to either run a Snowflake CLI command or some SQL to install the application in the account. 
```